### PR TITLE
Rebase Bluez 5.53 from Ubuntu repos

### PR DESCRIPTION
**Pull request by Flohack74**
Finally trying to execute a Bluez update we wanted to do 2 years ago.

- I based on @mariogrip ´s draft branch, looking at the failing patches, removing what was fixed up to 5.53 and refreshed a few of those CVE patches. Everything else applied cleanly, and CI seems to be happy. Now it needs testing.

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -21,7 +21,10 @@ CONFIGURE_FLAGS := \
 	--enable-systemd \
 	--enable-threads \
 	--enable-sixaxis \
-	--enable-experimental
+	--enable-experimental \
+	--enable-deprecated \
+	--enable-testing \
+	--enable-phonebook-ebook
```
Comment (mariogrip) [inline debian/rules]: This was removed here https://github.com/ubports/bluez-packaging/commit/fed7a89a7514612eb155ca1c9635faf140a5409d so i dont think we need it?

File: debian/patches/0003-tools-mpris-proxy-allow-user-to-specify-different-mp.patch [hunk 2]
```diff
@@ -1,65 +0,0 @@
-From 7abc94204c2dd72fd6a731a0240123e344dd2ac9 Mon Sep 17 00:00:00 2001
-From: Simon Fels <simon.fels@canonical.com>
-Date: Mon, 19 Oct 2015 22:13:02 +0200
-Subject: [PATCH 3/4] tools: mpris-proxy: allow user to specify different mpris
```
Comment (mariogrip) [inline debian/patches/0003-tools-mpris-proxy-allow-user-to-specify-different-mp.patch:4]: Is this not needed? is this a ubuntu touch thing or ubuntu desktop thing?

File: debian/patches/0004-obexd-enable-ebook-backend-conditionally.patch [hunk 3]
```diff
@@ -1,103 +0,0 @@
-From 5acde5532f16e1bb8b354baee00f8fe71fc38e52 Mon Sep 17 00:00:00 2001
-From: Simon Fels <simon.fels@canonical.com>
-Date: Fri, 11 Sep 2015 19:01:52 +0200
-Subject: [PATCH 4/4] obexd: enable ebook backend conditionally
```
Comment (mariogrip) [inline debian/patches/0004-obexd-enable-ebook-backend-conditionally.patch:4]: I guess we dont need eds anymore so i guess this is fine to be dropped.

File: debian/patches/0003-tools-mpris-proxy-allow-user-to-specify-different-mp.patch (hunk 2 above)
Comment (Flohack74) [reply debian/patches/0003-tools-mpris-proxy-allow-user-to-specify-different-mp.patch:4]: I have no idea what that is ^^

File: debian/rules (hunk 1 above)
Comment (Flohack74) [reply debian/rules]: Ouch yes will remove it

File: debian/patches/0004-obexd-enable-ebook-backend-conditionally.patch (hunk 3 above)
Comment (Flohack74) [reply debian/patches/0004-obexd-enable-ebook-backend-conditionally.patch:4]: alright!

File: debian/patches/0003-tools-mpris-proxy-allow-user-to-specify-different-mp.patch (hunk 2 above)
Comment (Flohack74) [reply debian/patches/0003-tools-mpris-proxy-allow-user-to-specify-different-mp.patch:4]: While I still have no idea this thing was not even in series, so deactivated already... SO I guess its fine to be removed.

**Review by mariogrip — APPROVED** [review]
LGTM :+1: lets get this in!

**Comment by lduboeuf** [comment]
Until around that update, i got that: https://github.com/ubports/ubuntu-touch/issues/2026

**Pull request merged**